config/v3_0_experimental/types/config.go

```diff
@@ -32,10 +30,7 @@ var (
 
 func (c Config) Validate() report.Report {
 	r := report.Report{}
-	rules := []rule{
-		checkFilesFilesystems,
-		checkDuplicateFilesystems,
-	}
+	rules := []rule{}
```

Comment:
This function now always returns `report.Report{}`, should the instantiation of `rules` and loop over nothing be dropped?

Reply:
I left the "infrastructure" for those checks there since we might want to add more, but I'm also ok removing it.